app/views/reviews/_review.html.erb

```diff
@@ -6,10 +6,10 @@
   <div>
     <h5 class="lang-review">
       <% review.english_rating.times do %>
-        <i class="fas fa-flag review-icon"></i>
```

Comment:
I already have done this part. I could have merged yours, but heroku does not accept the svg images if we do not specify extension